templates/python-project/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+console_output_style = progress
```

Comment:
❓ How sold are you on pytest as our standard testing framework? We're using it in hca because it's what I happened to reach for, but if there are substantive advantages to unittest we should use that instead.

Reply:
there are other third-party frameworks out there, but none that fill a major void in pytest's capabilities. on the contrary, unittest is a bit anemic compared to pytest (worse output formatting, missing some important utility features provided by pytest) as a test runner.

and, of course, we can just use all of unittest's features while still running our suite with pytest and getting all the benefits of its feature set.

Reply:
(most importantly, pytest is widely used enough that it has an active plugin ecosystem and rapid resolution time for bugs as new versions are released)